### ISSUE TITLE
Always get those errors when trying to use klc2ini.pl with the lates release 5.30.0.1 of strawberry perl portable for windows 

### ISSUE THREAD

**Issue by teddue** [Portable-Keyboard-Layout/Portable-Keyboard-Layout#17]
D:\Portable Keyboard Layout\utilities>perl klc2ini.pl "IPA.0.2.klc"
binmode() on closed filehandle INI at klc2ini.pl line 146.
print() on closed filehandle INI at klc2ini.pl line 147.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 173.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 173.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 175.
print() on closed filehandle INI at klc2ini.pl line 178.
print() on closed filehandle INI at klc2ini.pl line 179.
print() on closed filehandle INI at klc2ini.pl line 180.
print() on closed filehandle INI at klc2ini.pl line 181.
print() on closed filehandle INI at klc2ini.pl line 189.
print() on closed filehandle INI at klc2ini.pl line 190.
print() on closed filehandle INI at klc2ini.pl line 191.
print() on closed filehandle INI at klc2ini.pl line 192.
print() on closed filehandle INI at klc2ini.pl line 193.
print() on closed filehandle INI at klc2ini.pl line 194.

**Comment by rubo77** [comment]
Maybe the file is not writable?

**Comment by DreymaR** [comment]
I'm going to write up a in-program KLC-to-EPKL converter eventually. Then we can stop worrying about the intricacies of running Perl. Also, my converter will use a template .ini with a set of regular expressions in it, so you could write a converter file for PKL or from another format. But for now, it's nowhere near ready. Sorry.

**Comment by fzhwenzhou** [comment]
Try creating "out" folder in the same directory.

**Comment by DreymaR** [comment]
Meanwhile, you can use EPKL to generate in my opinion pretty help images without installing Perl. That does seem more convenient these days, doesn't it?

**Comment by fzhwenzhou** [comment]
> Meanwhile, you can use EPKL to generate in my opinion pretty help images without installing Perl. That does seem more convenient these days, doesn't it?

Don't know how to convert klc to EPKL layout.ini . I have used PKL for years and it works well on Windows 11.

**Comment by jarnosz** [comment]
> Try creating "out" folder in the same directory.

Indeed; all operations take place in the `out` directory, where the `ini` files and other stuff are generated.
